.github/workflows/validate_examples.yaml

```diff
@@ -88,6 +96,8 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version: ${{ env.GOVER }}
+        env:
+          GOPATH: ${{ github.workspace }}
```

Comment:
Is this really needed?

Reply:
Yeah it was failing without it with error `GoPath` not set